api/lib/mno_enterprise/intercom_events_listener.rb

```diff
@@ -0,0 +1,79 @@
+require 'intercom'
+
+module MnoEnterprise
+
+  class IntercomEventsListener
+
+    attr_accessor :intercom
+
+    def initialize
+      if MnoEnterprise.intercom_api_key && MnoEnterprise.intercom_api_secret
+        self.intercom = ::Intercom::Client.new(app_id: MnoEnterprise.intercom_api_key, api_key: MnoEnterprise.intercom_api_secret)
```

Comment:
A bit of mixup between the app id and the api_key (probably because of way the tenant_id work?)